skietbaan_fe/src/components/groups.css

```diff
@@ -120,47 +124,54 @@
   font-size: 16px;
   font-weight: lighter;
   padding-top: 10px;
+  position: fixed;
 }
+
 .userName {
   font-size: 16px;
   font-weight: bold;
   font-size: 16px;
   color: black;
 }
+
 .emails {
   font-size: 14px;
   font-weight: lighter;
   font-size: 16px;
   color: black;
 }
+
 .blabe {
   margin-left: 39px;
 }
-.theboxes{
+
+.theboxes {
   float: right;
 }
 
-.the-container{
+.the-container {
   width: 100%;
-  font-Weight: lighter;
+  font-weight: lighter;
   font-size: 18px;
   width: 100%;
-  color:white;
+  color: white;
 }
-.im{
+
+.im {
```

Comment:
These variables are not descriptive 
Also, inconsistent naming scheme either use 
`the-container` or `theBoxes` (note the camelCase)